modules/Administration/language/en_us.lang.php

```diff
@@ -912,5 +912,53 @@
     'LBL_PWDSEC_SPECCHAR_DESC' => 'Password should contain special characters',
     'LBL_PWDSEC_CHARS' => 'Characters',
 
+    // SearchWrapper
+    'LBL_SEARCH_HEADER' => 'Search',
+    'LBL_SEARCH_HEADER_DESC' => 'Manage your search preferences.',
+    'LBL_SEARCH_WRAPPER' => 'Search (New)',
+    'LBL_SEARCH_WRAPPER_DESC' => 'Manage your search preferences.',
+
+    // Elasticsearch Settings
+    'LBL_ELASTIC_SEARCH' => 'Elasticsearch',
+    'LBL_ELASTIC_SEARCH_SETTINGS' => 'Elasticsearch',
+    'LBL_ELASTIC_SEARCH_SETTINGS_DESC' => 'Configure Elasticsearch preferences.',
+    'LBL_ELASTIC_SEARCH_SETTINGS_HELP' => 'Elasticsearch requires an external server set up to work. Read the <a href="https://docs.suitecrm.com/admin/administration-panel/search/elasticsearch/">documentation</a> to learn how.',
+    'LBL_ELASTIC_SEARCH_GENERAL' => 'General',
+    'LBL_ELASTIC_SEARCH_ENABLE' => 'Enable Elasticsearch.',
+    'LBL_ELASTIC_SEARCH_TEST_CONNECTION' => 'Test connection',
+    'LBL_ELASTIC_SEARCH_TEST_CONNECTION_SUCCESS' => 'Connection successful.',
+    'LBL_ELASTIC_SEARCH_TEST_CONNECTION_FAIL' => 'Connection failed.',
+    'LBL_ELASTIC_SEARCH_TEST_CONNECTION_ERROR' => 'Failed perform ping request.',
+    'LBL_ELASTIC_SEARCH_SERVER' => 'Server',
+    'LBL_ELASTIC_SEARCH_HOST' => 'Host',
+    'LBL_ELASTIC_SEARCH_USER' => 'Username',
+    'LBL_ELASTIC_SEARCH_PASS' => 'Password',
+    'LBL_ELASTIC_SEARCH_SCHEDULERS' => 'Schedulers',
+    'LBL_ELASTIC_SEARCH_SCHEDULERS_HELP' => 'The following ElasticSearch schedulers were detected:',
+    'LBL_ELASTIC_SEARCH_SCHEDULERS_LAST_RUN' => 'last run:',
+    'LBL_ELASTIC_SEARCH_SCHEDULERS_NEVER_RUN' => 'This job has never run',
+    'LBL_ELASTIC_SEARCH_SCHEDULERS_NOT_FOUND' => 'No schedulers found. Consider creating one.',
+    'LBL_ELASTIC_SEARCH_SCHEDULERS_DESC' => 'The ElasticSearch module uses schedulers to keep the database and the indexing engine synchronised.',
+    'LBL_ELASTIC_SEARCH_INDEX' => 'Index',
+    'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_HELP' => 'Schedule an indexing.',
+    'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_FULL' => 'Schedule full indexing',
+    'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_PART' => 'Schedule partial indexing',
+    'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_FULL_SUCCESS' => 'A full indexing has been scheduled and will start in the next 60 seconds. Search results might be incomplete until the process is complete.',
+    'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_FULL_FAIL' => 'Failed to start full index.',
+    'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_PART_SUCCESS' => 'A full indexing has been scheduled and will start in the next 60 seconds.',
```

Comment:
Typo in the 'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_PART_SUCCESS' value. 
Replace please 'A full indexing' with 'A partial indexing'.

Reply:
Thank you for pointing that out! :+1: